library/Core/src/Juvix/Core/Erasure/Algorithm.hs

```diff
@@ -153,11 +152,11 @@ piTypeToList ty =
       let (rest, res) = piTypeToList ret in ((usage, arg) : rest, res)
     _ -> ([], ty)
 
-listToPiType ::
-  ([(Usage.Usage, IR.Term primTy primVal)], IR.Term primTy primVal) ->
-  IR.Term primTy primVal
-listToPiType ([], ret) = ret
-listToPiType ((u, x) : xs, ret) = IR.Pi u x (listToPiType (xs, ret))
+-- listToPiType ::
```

Comment:
Not called anywhere, except mentioned above. Can we delete these lines if there are no future plans?

Reply:
If there are no future plans, please delete all commented out lines before merging this. Thanks!